spec/spec_helper.rb

```diff
@@ -1,3 +1,5 @@
+$LOAD_PATH.unshift('./lib/')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.